api/v6/products.thrift

```diff
@@ -31,13 +31,14 @@ typedef list<ProductConfiguration> ProductConfigurations
 
 /* Product carries data to the end user's product list and tasks. */
 struct Product {
-  1: i64    id,
-  2: string endpoint,
-  3: string displayedName_b64,
-  4: string description_b64,
-  5: bool   connected,         // Indicates that the server could set up the database connection properly.
-  6: bool   accessible,        // Indicates whether the current user can access this product.
-  7: bool   administrating     // Indicates that the current user can administrate the product.
+  1: i64               id,
+  2: string            endpoint,
+  3: string            displayedName_b64,
+  4: string            description_b64,
+  5: bool              connected,         // Indicates that the server could set up the database connection properly.
```

Comment:
Is this attribute still relevant if `databaseStatus` conveys such values?

Reply:
I do not wanted to break the api in a backward compatible way, but yes this value is not needed.

Reply:
Understandable. Though, I'd rather have a comment here at least about the status of this variable.